### DISCUSSION TITLE
How to allow ordering only to authorized users?

### DISCUSSION THREAD

**Discussion by yellotra** [OFFLINE-GmbH/oc-mall-plugin#1034] [Q&A]
Now it is possible to make an order within a session for users who are not logged in

**Comment by tobias-kuendig** [comment]
Can you provide a reproduction for this problem? There are multiple events you can use to abort a order before it is created:

https://offline-gmbh.github.io/oc-mall-plugin/development/events.html#mall-order-beforecreate
